### ISSUE TITLE
Re-importing an existing book places a msg on the earliest history item instead of the 'import existing book' entry

### ISSUE THREAD

**Issue by hornc** [internetarchive/openlibrary#1381]
Initial record created, from an amazon.com record. is appearing on the first edit on a book's history footer, when it really should be associated with the re-import change.

Example:
https://openlibrary.org/books/OL26354767M/How_to_Be_a_Stoic

**Comment by tfmorris** [comment]
~That example doesn't appear to match the problem description.~ It looks like the original record was created by a user, not from an Amazon record, unless the edit history is wrong. The only thing the latest edit appears to have changed was to add an Amazon ID as a source record, even though no other metadata was changed, making it not really a source for anything.

I suggest that source records not be added unless they are actually the source for something. I also suggest that if a source is used, it be linked in the edit history comment so that one can click through to it (for both MARC and Amazon records).

Of course, we should get this all sorted out before doing any bulk imports, so it's a blocker for those.

**Comment by hornc** [comment]
I don't believe this is a blocker, it's just a UI display issue affecting the History template, which is misleading. The underlying metadata stored on the item is correct, and the specific history diffs at every stage are correct.

I believe  https://openlibrary.org/books/OL26354767M/How_to_Be_a_Stoic is a valid example of the problem as the history template summary states: "Created by <username> | Inital record created, from an amazon.com record."

but the Amazon.com record was only added very recently, by the last edit https://openlibrary.org/books/OL26354767M/How_to_Be_a_Stoic?b=4&a=3&_compare=Compare&m=diff

The "Initial record created" msg is added by the template which assumes the presence of a source means it was initially created from it, which is not the case here as source_records can be added later.   `source_records` is a list, which suggests that multiple metadata sources can contribute to one OL record. I'd treat the no-other current change edit as a confirming source, and gives us the potential to verify or obtain more data later if we we need to.

**Comment by jdlrobson** [comment]
Not sure I understand this one as not seeing any mention of amazon here:

![Screenshot 2019-08-10 at 10 17 02 AM](https://user-images.githubusercontent.com/148752/62824839-0d875580-bb58-11e9-84bb-076bedae5b4a.png)

**Comment by guyjeangilles** [comment]
@hornc WIll importbot changes affect/fix this?

**Comment by hornc** [comment]
The history also appears different depending on whether a user is logged in or not. To see the incorrect overwirtten history you have to be logged in.

**Comment by cdrini** [comment]
Not sure if this was closed intentionally while closing the duplicate! This is an issue @seabelis brings up as making librarian work difficult, so it is still a bug we should target when we get the opportunity.
